.github/dependabot.yml

```diff
@@ -20,6 +20,10 @@ updates:
   groups:
     all-github-actions:
```

Comment:
I guess we should change this like the rest to `github-actions`?

Reply:
Good catch, missed it. Fixed.